src/Nox.Generator/Domain/EntityMeta.template.cs

```diff
@@ -14,11 +14,30 @@ namespace {{codeGeneratorState.DomainNameSpace}};
 /// </summary>
 public partial class {{className}}
 {
-{{- for entry in propertiesWithSource }}
+{{- for td in typeDef }}
+    {{ if td.OptionsOutput != "" }}
     /// <summary>
-    /// Type options and factory for property '{{entry.TypeDef.Name}}'
+    /// Type options for property '{{td.Name}}'
     /// </summary>
-    {{entry.Source}}
+    public static {{td.OptionsOutput}}
```

Comment:
the refactor of the template it to avoid this kind of code and be explicitly on the template what code we are generating